.github/workflows/markdown_format.yml

```diff
@@ -0,0 +1,20 @@
+name: markdown formatting
+
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '**/*'
```

Comment:
**issue (blocking):** It could be run on pull request and if it contains markdown files

```yaml
on:
  pull_request:
    paths:
    - '**.md'

```
something like that